test/DatabaseIntegrationTest/Tools/SQLExecutor.cpp

```diff
@@ -0,0 +1,93 @@
+#include "stdafx.h"
+#include "SQLExecutor.h"
+
+#include <fstream>
+#include <sqlite3.h>
+#include <stdio.h>
+#include <boost/filesystem.hpp>
+
+
+namespace seed_cpp { namespace db_test {
+
+	SQLExecutor::SQLExecutor(const std::string& dbFilepath)
+		:m_dbFilepath(dbFilepath)
+	{
+	}
+
+	SQLExecutor::~SQLExecutor() = default;
+
+	bool SQLExecutor::executeScripts(const std::vector<std::string>& scripts)
+	{
+		for (const auto& script : scripts)
+		{
+			if (!executeScript(script))
+			{
+				return false;
+			}
+		}
+
+		return true;
+	}
+
+	bool SQLExecutor::executeScript(const std::string& scriptFilepath)
+	{
+		std::ifstream ifs(scriptFilepath);
+		if (!ifs)
+		{
+			std::cout << "SQLExecutor::executeScript() Unable to find script '"
+					  << boost::filesystem::absolute(boost::filesystem::path(scriptFilepath))
+					  << std::endl;
+			return false;
+		}
+
+		std::string scriptContents;
+		ifs.seekg(0, std::ios::end);
+		scriptContents.resize(static_cast<unsigned int>(ifs.tellg()));
```

Comment:
You could change this to work like in Core for the database schema creation

Reply:
Updated to improve it as suggested